jax/_src/scipy/special.py

```diff
@@ -1719,3 +1719,195 @@ def bernoulli(n: int) -> Array:
   k = jnp.arange(2, 50, dtype=bn.dtype)  # Choose 50 because 2 ** -50 < 1E-15
   q2 = jnp.sum(k[:, None] ** -m[None, :], axis=0)
   return bn.at[4::2].set(q1 * (1 + q2))
+
+
+@custom_derivatives.custom_jvp
+@jnp.vectorize
+@_wraps(osp_special.poch, module='scipy.special', lax_description="""\
+The JAX version only accepts positive and real inputs.""")
+@jit
```

Comment:
`jit(vectorize)` would be better than `vectorize(jit)` if you want to jit-compile the function by default.